lambda_function/config.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, REQUEST_ORIGIN: str, REQUEST_PATH: str, ID: str = None):
                         {
                             'API_URL': 'https://api2.geode-solutions.com',
                             'ASSIGN_PUBLIC_IP': 'ENABLED', 'CLUSTER_NAME': 'C_ShareTwin_Prod', 'ENVIRONMENT_VARIABLES': {
-                                'name': 'Geode', 'environment': [{
```

Comment:
are you sure ?

Reply:
Yes, I renamed all the containers in the task definition, because at first I thought I had to do a proxy_pass http://geode:5000 with nginx, but routing in fargate works with localhost.
In the end it's good like that